ios/brave-ios/Sources/AIChat/ModelView/AIChatViewModel.swift

```diff
@@ -68,11 +68,11 @@ public class AIChatViewModel: NSObject, ObservableObject {
 
   public var shouldShowPremiumPrompt: Bool {
     get {
-      return canShowPremiumPrompt
+      return _canShowPremiumPrompt
     }
 
     set {  // swiftlint:disable:this unused_setter_value
-      self.canShowPremiumPrompt = newValue
+      _canShowPremiumPrompt = newValue
```

Comment:
@Brandon-T we don't need to call `objectWillChange.send()` right?

Reply:
Correct. Not necessary.